src/main/java/sirius/biz/tenants/TenantData.java

```diff
@@ -211,10 +217,36 @@
 
     /**
      * Contains the features and individual config assigned to the tenant.
+     *
+     * @deprecated use {@link ::packageData}
```

Comment:
why not drop this alltogether and mark it a new major release - this won't be updated anyway(?)